### PR TITLE
Transfer mongodb cookbook on Supermarket to Sous Chefs org

### PR DESCRIPTION
**Pull request by shortdudey123**

### CODE REVIEW

**Comment by nathenharvey** [comment]
@shortdudey123  please make sure the RFC lists the reasons why using the current cookbook could be harmful.

>People currently using the cookbook on supermarket are getting an EOS'd version and is not receiving security updates

**Comment by nathenharvey** [comment]
This PR will go on hold until March 15, 2017.  During this time the following will happen:

1.  @nathenharvey will reach out to @thekorn weekly asking for a decision on the ownership of the cookbook.
1.  The Sous Chefs will publish their MongoDB cookbook to the Supermarket.
1.  On, or after, March 15, 2017, a Supermarket Administrator will mark the current MongoDB cookbook as deprecated and offer the Sous Chefs' cookbook as a suitable replacement.
1.  A message will be posted to the mailing list notifying of this change and our reasoning behind it.

As the current owner of the cookbook, @thekorn can accelerate or stop this timeline at any time.

File: new/mongodb_cookbook_transfer.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+maintainer was by [jamesonjlee](https://github.com/edelight/chef-mongodb/pull/356)
+on December 9, 2014.
+
+The Sous Chefs have been maintaining for [fork](https://github.com/sous-chefs/mongodb)
+of the mongodb cookbook.  The fork's master branch is currently 90 commits ahead
+of the source repository for the cookbook. The Sous Chefs have been active on the issues
+on the origional repository and have worked to fix them in the maintained fork.
+
+Grant Ridder, on behalf of the Sous Chefs, has attempted on several ocasaions during
+November and December of 2016 to contact Markus Korn, the current ownwer on Supermarket,
```
Comment (mburns) [inline new/mongodb_cookbook_transfer.md]: `s/ownwer/owner/`

File: new/mongodb_cookbook_transfer.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+Currently the [mongodb cookbook](https://github.com/edelight/chef-mongodb)
+on [Supermarket](https://supermarket.chef.io/cookbooks/mongodb) was last released
+on November 11, 2014.  The last commit on the GitHub repository was the day prior,
+November 10, 2014.  At the time of the writing of this draft, the repository has
+70 open issues and 35 open PRs.  The last comment on any issue or PR by any
+maintainer was by [jamesonjlee](https://github.com/edelight/chef-mongodb/pull/356)
+on December 9, 2014.
+
+The Sous Chefs have been maintaining for [fork](https://github.com/sous-chefs/mongodb)
```
Comment (mburns) [inline new/mongodb_cookbook_transfer.md]: `s/for/a/` ?

**Comment by shortdudey123** [comment]
@nathenharvey i have updated the RFC with the feedback so that it can be discussed and finalized this week at the dev meeting since it is now passed the hold date, thanks!

**Comment by nathenharvey** [comment]
[Discourse post](https://discourse.chef.io/t/deprecating-the-mongodb-cookbook-please-use-sc-mongodb/10636) has been made.

[MongoDB cookbook](https://supermarket.chef.io/cookbooks/mongodb) is now deprecated and suggests the [sc-mongodb cookbook](https://supermarket.chef.io/cookbooks/sc-mongodb) as a suitable alternative.

This PR should now be ready for merging.

**Comment by thommay** [comment]
@chef/rfc-editors please merge this!

**Comment by adamleff** [comment]
Squashed, RFC088 assigned, merged in 945a07b. Thanks, @shortdudey123!

**Pull request closed without merging**